model/auth.go

```diff
@@ -99,15 +100,35 @@ func (a *AuthDefinitions) UnmarshalJSON(b []byte) error {
 		return fmt.Errorf("no bytes to unmarshal")
 	}
 
-	// See if we can guess based on the first character
-	switch b[0] {
-	case '"':
-		return a.unmarshalFile(b)
-	case '[':
-		return a.unmarshalMany(b)
-	}
+	for index, char := range b {
+
+		switch {
+		case b[index] >= '0' && b[index] <= '9':
+			fmt.Print(string(char))
```

Comment:
We shoule delete this.

Reply:
maybe, this makes sure that numbers can't be parsed, as you've asked before :)